client/app/controllers/collection/add.js

```diff
@@ -4,8 +4,7 @@ export default Ember.Controller.extend({
     addMethod: 'select', // 'select' or 'create'
     methodSelected: false,
     type: Ember.computed('model.settings', function(){
-        let type = this.get('model.settings.collectionType').toLowerCase();
-        return  type === 'preprint' || type === 'project' ? 'node' : type;
+        return this.get('model.settings.collectionType').toLowerCase();
```

Comment:
Yes project works. CollectionType would be required but users can delete it from the settings field in edit right now, which will not be the ui that we use for it in the future.